package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/source-deploy-retrieve",
-  "version": "3.1.1",
+  "version": "4.0.0",
   "gusBuild": "49.1.0",
```

Comment:
does gusBuild have to be updated?

Reply:
do you know where we use it? is it for git2gus?

Reply:
i'm not sure where it's used